### PR TITLE
HPCC-16676 Implement ConfigMgr Session Management

### PR DESCRIPTION
**Pull request by RussWhitehead**
Add new AuthDomain tab to Config Manager ESP config.  This dynamic table
captures 0 or more entries that define an ESP authorization domain(s). Each
entry capture DomainName, Auth Type, Login/logoutURL, resource URL and
session  duration

Signed-off-by: Russ Whitehead <william.whitehead@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [x] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by RussWhitehead** [comment]
@wangkx Please review

File: initfiles/componentfiles/configxml/esp.xsd.in [hunk 1]
```diff
@@ -347,6 +347,86 @@
...
+	              </xs:annotation>
+	            </xs:attribute>
+	            <xs:attribute name="domainName" type="xs:string" use="required" default="">
+	              <xs:annotation>
+	                <xs:appinfo>
+	                  <tooltip>Unique string to identify an auth domain in case > 1 domains in an esp</tooltip>
+	                  <colIndex>2</colIndex>
+	                </xs:appinfo>
+	              </xs:annotation>
+	            </xs:attribute>
```
Comment (wangkx) [inline initfiles/componentfiles/configxml/esp.xsd.in]: Can we make 'name' not readonly and merge 'domainName' into it?

**Review by wangkx — COMMENTED** [review]
@RussWhitehead  1 comment

**Comment by RussWhitehead** [comment]
@wangkx Please rereview

**Comment by wangkx** [comment]
@RussWhitehead looks fine.

**Comment by RussWhitehead** [comment]
@richardkchapman Please merge

**Pull request merged**